### ISSUE TITLE
what changed in v3? / changelog?

### ISSUE THREAD

**Issue by ljharb** [joeattardi/json-colorizer#37]
I can't tell what the breaking changes are in v3, and there's no changelog. Can you help me understand how I can safely upgrade?

**Comment by joeattardi** [comment]
Hey there, sorry for not including a changelog. Mea culpa.

Here is an overview of what changes for v3.

- When setting custom colors, token type names are now `PascalCase` instead of `SCREAMING_CASE`.
- When specifying the colors, instead of using a string for the color function, you need to import the `color` object exported by the library. This contains named functions for each main color (this change was because I changed the underlying color library used).
- The `pretty` option has been replaced with an `indent` option. By default the library pretty prints the JSON with indentation of 2 spaces. If you don't want pretty printing, you can set `indent` to `0`.

Other than that the core usage of the library is the same. You can pass an object or a JSON string to the `colorize` function. 

I hope this helps, and my apologies for the confusion.

**Comment by ljharb** [comment]
Thanks! I’ll try to migrate and see how it goes.

Does indent also take a tab character?

**Comment by joeattardi** [comment]
No, just a number of spaces. It just passes that option through as the third argument to `JSON.stringify`.

**Comment by ljharb** [comment]
JSON.stringify takes a string also, so that tabs can be used.

**Comment by joeattardi** [comment]
Really? I never knew that!

**Comment by ljharb** [comment]
Looks like another change is that the `module.exports` value is no longer the colorize function itself, so i need to `.colorize` to get at the function.
